sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -387,9 +387,13 @@ export class ApplicationService extends RecordDataModelService<Application> {
     const auditUser = { id: auditUserId } as User;
     const originalAssessment = new StudentAssessment();
     originalAssessment.triggerType = AssessmentTriggerType.OriginalAssessment;
+    originalAssessment.studentAssessmentStatus =
+      StudentAssessmentStatus.Submitted;
```

Comment:
What is the reason for explicitly setting this status?

Reply:
The idea was to have the date submitted aligned to also use the `now`, and it seems not clear to set the date without setting the status. I can remove it if it would cause more confusion but I would also remove the date then and adjust the E2Es.

Reply:
That's alright. I wasn't sure about the reason and so made the comment.